src/controllers/room.controller.js

```diff
@@ -65,6 +62,15 @@ const create = async function(req, res) {
         if(req.body.is_private) {
             is_private = req.body.is_private;
         }
+        if(req.body.authorised_users) {
+            authorised_users = req.body.authorised_users;
+            if(!is_private && authorised_users.length!=0){
+                console.log('Error. No se puede crear una sala no privada y añadir gente autorizada. ' + err);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Backend/pullRequest?prid=7196401)
```suggestion
                console.log("Error. No se puede crear una sala no privada y añadir gente autorizada. " + err);
```